content/blog/polkadot-generative-drops-on-kodadot.md

```diff
@@ -0,0 +1,85 @@
+
+Previous Polkadot Drops In the last few weeks, we’ve done another three of these free drops, and let me tell you, it’s been a blast! People are loving it. And guess what? We’re not stopping. We’re planning to do even more drops and give people a proper collecting experience.
+
+### 3) Swirls
+
+Artist: [Sabha](https://twitter.com/sabhab1)
+
+<img src="/blog/polkadot-generative-drops-on-kodadot-4.webp" alt="Swirls collection" />
+
+Check the collection [HERE](https://kodadot.xyz/ahp/collection/40).
+
+Swirls are mesmerizing art pieces. Each of them is full of dots \*I guess we all love Polkadot( Each piece in the collection is born from a generative algorithm that mirrors the spiraling sequences found in galaxies and growth rings, revealing the underlying order in apparent randomness. The collection’s palette is a tribute to these organic forms, with colors ranging from earthy and subtle to vibrant and dynamic, reflecting the diverse expressions of natural swirls.
+
+In this artistic endeavor, the algorithm acts as a bridge between the digital realm and the recursive beauty of the natural world. Collectors are invited to own a piece of this perpetual motion, where each NFT is a snapshot of creation, a harmonious blend of art and mathematics that resonates with the rhythm of life’s designs.
+
+### 4) Snowflakes
+
+Artist: [Aurora](https://twitter.com/poppyseedDev)
+
+<img src="/blog/polkadot-generative-drops-on-kodadot-5.webp" alt="Snowflakes collection" />
+
+Every snowflake must go through “Fractal Quest,” which consists of five quests that will create a whole story about these NFTs.
+
+Check the collection [HERE](https://kodadot.xyz/ahp/collection/46).
+
+### 5) Pare1d0scope
+
+Artist: [sha22.art](http://sha22.art/)
+
+<img src="/blog/polkadot-generative-drops-on-kodadot-6.webp" alt="Pare1d0scope collection" />
+
+Check the collection [HERE](https://kodadot.xyz/ahp/collection/38).
+
+This project is a collaboration between [sha22.art](http://sha22.art/), Antonino, and KodaDot. Anchored in a not-so-binary world, the pare1d0scope generates unique patterns of 4-bit squares by applying a generative mix of symmetry, ordering, subdividing, and diagonal adjustments. This collection features a special color palette to celebrate Generative Art’s landing in the Polkadot space! Play with the NFTs even after minting; each NFT is interactive!
+
+## What's next?
+
+Collect the NFTs because we are preparing for you a collecting experience. Some of the drops will be rarer than others, and only those holders who own NFT from previous collections will be able to mint it.
+
+## Why should I list my NFT for sale?
+
+- Profit Potential: The primary reason for listing NFTs soon after purchase is the potential for profit. If the NFT has gained popularity or increased in value since its acquisition, selling it quickly can yield a significant return on investment.
```

Comment:
maybe remove all "-" in this paragraph. It doesn't look good published. Maybe do it in bold:

**Market Trends and Hype**: The value of NFTs can be highly dependent on current trends and hype. Listing them while they are still in high demand ensures that the seller capitalizes on these market sentiments.
**Volume growth**: Listing an NFT soon after purchase can contribute to the overall activity and health of the NFT collection. And with that, we all benefit.
**Experimentation and Learning**: For newcomers to the NFT space, quick buying and selling can serve as a learning experience, helping them understand market dynamics without committing to long-term investments.
**Traders will be rewarded too:** We encourage you to trade your newly obtained NFTs. List it for sale, or buy a new one, and contribute to the growing volume of the collection. Remember, traders will be rewarded on KodaDot.